maptemplate.c

```diff
@@ -1310,6 +1316,15 @@ static int processItemTag(layerObj *layer, char **line, shapeObj *shape)
       tagValue = msReplaceSubstring(tagValue, "$value", itemValue);
       msFree(itemValue);
 
+      if (padding != -1) {
+          int paddedSize = sizeof(tagValue) + (padding * sizeof(char*));
+          char *paddedValue = NULL;
+          paddedValue = (char *) msSmallMalloc(paddedSize + 1);
+          snprintf(paddedValue, paddedSize, paddingFormatString, padding, tagValue);
```

Comment:
for clarity, remove paddingFormatString and replace it by its inline value paddingFormatString